build/templates/errors.rst.mako

```diff
@@ -80,6 +82,17 @@ ${helper.get_rst_header_snippet('SelfTestError', '-')}
         An error due to a failed self-test
 
 
+% endif
+% if grpc_supported:
+${helper.get_rst_header_snippet('RpcError', '-')}
+
+    .. py:currentmodule:: ${module_name}.errors
+
+    .. exception:: RpcError
+
+        An error specific to gRPC sessions
```

Comment:
is this what we decided to call these? "gRPC sessions"?
Want to make sure we use consistent terminology across the MeasurementLink board.

Reply:
"An error specific to sessions to the NI gRPC Device Server"